File: matrix001.cpp
#include <bits/stdc++.h>

int main() {
    int row, col;
    int m1[100][100], m2[100][100];
    
    std::cout << "Please input row (m): ";
    std::cin >> row;
    
    std::cout << "Please input column (n): ";
    std::cin >> col;
    
    for(int i = 0; i < row; i++){
        for(int j = 0; j < col; j++){
            std::cout << "Matrix 1 [" << i+1 << "][" << j+1 << "]: ";
            std::cin >> m1[i][j];
        }
    }
    
    for(int i = 0; i < row; i++){
        for(int j = 0; j < col; j++){
            std::cout << "Matrix 2 [" << i+1 << "][" << j+1 << "]: ";
            std::cin >> m2[i][j];
        }
    }
    
    std::cout << "Matrix Addition" << std::endl;
    for(int i = 0; i < row; i++){
        for(int j = 0; j < col; j++){
            std::cout << m1[i][j] + m2[i][j] << "\t";
        }
        std::cout << std::endl;
    }
    
    std::cout << std::endl;
    
    std::cout << "Matrix Subtraction" << std::endl;
    for(int i = 0; i < row; i++){
        for(int j = 0; j < col; j++){
            std::cout << m1[i][j] - m2[i][j] << "\t";
        }
        std::cout << std::endl;
    }
}

File: matrix001.py
m1 = []
m2 = []

row = int(input("Please input row (m): "))
col = int(input("Please input column (n): "))

for i in range(row):
  arr = []
  for j in range(col):
    n = int(input("Matrix 1 [%d][%d]: " %(i+1, j+1)))
    arr.append(n)
  m1.append(arr)

for i in range(row):
  arr = []
  for j in range(col):
    n = int(input("Matrix 2 [%d][%d]: " %(i+1, j+1)))
    arr.append(n)
  m2.append(arr)

for i in range(len(m1)):
    for j in range(len(m1[0])):
        print(m1[i][j] + m2[i][j], end='\t')
    print("\n")
